tests/setup-test-env.py

```diff
@@ -32,6 +32,7 @@
 test_frontend_config_path = os.path.join(current_dir, TEST_FRONTEND_CONFIG_NAME)
 
 DB_PW = 'testpw'
+TZ_VALUE = 'Europe/Berlin'
```

Comment:
The timezone is now fixed to Europe/Berlin for tests.

What is the rationale here?

IMHO I would say that having no timezone information, which will default then to UTC, is better than having it set to an arbitrary location. If you run the tool in Brazil you will be quite confused.

Why not make the same inference with default settings as it is done in the install script?